Easy/1. Shopping Cart Problem/solutions/README.md

```diff
@@ -1 +1,11 @@
 # Solutions
+Language Used : C++
+BASIC APPROACH 🔢 to solve the program :
+
+-> Used class in program 
+
+-> Created object in main function to access class members functions.
+
+-> Used Time library (<ctime>) to get Real Date , Time , Year and day of user current usage during program.
+
+-> Added little bit systematic styles to program for better view to user and easy understandings of the program to user.
```

Comment:
Please remove this change and add the solution approach inside your own folder (i.e. `Anmolreshi_anmolreshi`).

Reply:
okay